src/app/app.module.ts

```diff
@@ -1,5 +1,5 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule, APP_INITIALIZER, DoBootstrap, ApplicationRef } from '@angular/core';
+import { NgModule, APP_INITIALIZER, DoBootstrap, CUSTOM_ELEMENTS_SCHEMA, ApplicationRef } from '@angular/core';
```

Comment:
remove CUSTOM_ELEMENTS_SCHEMA if not required

Reply:
removed